### DISCUSSION TITLE
Get @apiParam allowed values from variable

### DISCUSSION THREAD

**Discussion by fabrizzio-gz** [apidoc/apidoc#1394] [Ideas]
Example:

```javascript
const ALLOWED_VALUES = [1, 2, 3];

/**
 * @apiParam {number=${ALLOWED_VALUES}} [numbers]
 */
```

And have that parsed in the Apidoc as parameter `numbers` with allowed values `1`, `2`, and `3`.

Of course, any other option besides `${var}` to pass a variable to the inline documentation would be fine.

**Comment by NicolasCARPi** [comment]
But that would mean that the source code should also be parsed and understood by the apidoc parser!
